example-junit5/src/test/java/com/epam/reportportal/example/junit5/nested/NestedSteps.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.reportportal.example.junit5.nested;
+
+import com.epam.reportportal.annotations.Step;
+import com.epam.reportportal.junit5.ReportPortalExtension;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
```

Comment:
Please add some explanation what the test does.

Reply:
Description provided